chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,330 @@
+# Pre-trained models and datasets for text-to-speech
+
+Text-to-speech task (also called _speech synthesis_) comes with a range of challenges.
+
+First, just like in the previously discussed automatic speech recognition, the alignment between text and speech can be tricky.  
+However, unlike ASR, TTS is a one-to-many mapping problem, i.e. the same text can be synthesised in many different ways. Even different outputs (spectrograms or audio waveforms) can correspond to the same ground truth. The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
+especially for long and complex sentences.
+
+Next, there's the long-distance dependency problem: language has a temporal aspect, and understanding the meaning of a 
+sentence often requires considering the context of surrounding words. Ensuring that the TTS model captures and retains 
+contextual information over long sequences is crucial for generating coherent and natural-sounding speech.
+
+Finally, training TTS models typically requires pairs of text and corresponding speech recordings. On top of that, to ensure 
+the model can generate speech that sounds natural for various speakers and speaking styles, data should contain diverse and 
+representative speech samples from multiple speakers. Collecting such data is expensive, time-consuming and for some languages 
+is not feasible. You may think, why not just take a dataset designed for ASR (automatic speech recognition) and use it for 
+training a TTS model? Unfortunately, automated speech recognition (ASR) datasets are not the best option. The features that 
+make it beneficial for ASR, such as excessive background noise, are typically undesirable in TTS. It's great to be able to 
+pick out speach from a noisy street recording, but not so much if your voice assistant replies to you with cars honking 
+and construction going full-swing in the background. Still, some ASR datasets can sometimes be useful for fine-tuning, 
+as finding top-quality, multilingual, and multi-speaker TTS datasets can be quite challenging.
+
+Let's explore a few datasets suitable for TTS that you can find on the 🤗 Hub.
+
+## TTS Datasets 
```

Comment:
This is super nice! IMO worthy of its own section - would probably split this section up into datasets + pre-trained models (if you feel you have the time)